From 46ba4853753a4b173d4f74c6b89cf3c1fe0968ca Mon Sep 17 00:00:00 2001
From: Peter Williams <peter@newton.cx>
Date: Tue, 11 Sep 2018 12:50:30 -0400
Subject: [PATCH 1/5] Disable test bz106632

It fails on macOS and Windows.

diff --git a/test/Makefile.am b/test/Makefile.am
index aea8724..99b5387 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -109,20 +109,6 @@ TESTS += run-test-conf.sh
 check_PROGRAMS += test-bz106618
 test_bz106618_LDADD = $(top_builddir)/src/libfontconfig.la
 
-if !OS_WIN32
-check_PROGRAMS += test-bz106632
-test_bz106632_CFLAGS =					\
-	-I$(top_builddir)				\
-	-I$(top_builddir)/src				\
-	-I$(top_srcdir)					\
-	-I$(top_srcdir)/src				\
-	-DFONTFILE='"$(abs_top_srcdir)/test/4x6.pcf"'	\
-	-DHAVE_CONFIG_H					\
-	$(NULL)
-test_bz106632_LDADD = $(top_builddir)/src/libfontconfig.la
-TESTS += test-bz106632
-endif
-
 check_PROGRAMS += test-issue107
 test_issue107_LDADD =					\
 	$(top_builddir)/src/libfontconfig.la		\
