From ed82c63c43018dc8c1270f12f6d667bafd232e4d Mon Sep 17 00:00:00 2001
From: Peter Williams <peter@newton.cx>
Date: Tue, 11 Sep 2018 12:51:29 -0400
Subject: [PATCH 2/5] Disable some more Windows-unfriendly tests.

diff --git a/test/Makefile.am b/test/Makefile.am
index 99b5387..dd129a6 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -38,7 +38,7 @@ SH_LOG_COMPILER = sh
 if OS_WIN32
 LOG_COMPILER = ${srcdir}/wrapper-script.sh
 endif
-TESTS=run-test.sh
+TESTS=
 
 TESTDATA =			\
 	4x6.pcf			\
